patroni/postgresql/config.py

```diff
@@ -300,28 +315,26 @@ def write_postgresql_conf(self, configuration=None):
         if 'custom_conf' not in self._config and not os.path.exists(self._postgresql_base_conf):
             os.rename(self._postgresql_conf, self._postgresql_base_conf)
 
-        with open(self._postgresql_conf, 'w') as f:
-            os.chmod(self._postgresql_conf, stat.S_IWRITE | stat.S_IREAD)
-            f.write(self._CONFIG_WARNING_HEADER)
-            f.write("include '{0}'\n\n".format(self._config.get('custom_conf') or self._postgresql_base_conf_name))
+        with ConfigWriter(self._postgresql_conf) as f:
+            include = self._config.get('custom_conf') or self._postgresql_base_conf_name
+            f.writeline("include '{0}'\n".format(ConfigWriter.escape(include)))
```

Comment:
There is already one `\n` inside `writeline` method.

Reply:
Originally there were two: https://github.com/zalando/patroni/pull/1194/files#diff-a534a7e0f2c46ba53349f820d9014468L306, therefore I am adding the second one.